php/libraries/Utility.class.inc

```diff
@@ -983,37 +798,6 @@ class Utility
         return $test_name;
     }
 
-    /**
-     * Get all the sourcefrom instruments from parameter_type (which
-     * exist in the data_integrity flag table)
-     *
-     * @return  Associative array of the form array('Full Name' => 'Full Name')
-     * @note    This should be moved out of the Utility class into whatever
-     *       module uses it. (Data team helper?)
-     * @note    Function comment written by Dave, not the author of this function.
-     * @cleanup
-     */
```

Comment:
same...
IBIS redefines this function in its ProjectUtility class but neither functions are used anywhere in the code

@sruthymathew123 potential clean-up for you !

Reply:
I think this was used by the really old "quat table" DQT. It's unused, and IBIS ProjectUtility probably just copied the Utility to start with.